extensions/network/net-systemd-neworkd.sh

```diff
@@ -0,0 +1,31 @@
+#
+# Extension to manage network interfaces with systemd-networkd + Netplan
+#
+function add_host_dependencies__install_systemd_networkd() {
```

Comment:
`add_host_dependencies` is not adequate for this. use `extension_prepare_config` or such.